lib/broccoli/ember-app.js

```diff
@@ -617,6 +619,41 @@ class EmberApp {
     }, []);
   }
 
+  _compileAddonTemplates(tree) {
+    if (registryHasPreprocessor(this.registry, 'template')) {
+      tree = preprocessTemplates(tree, {
+        annotation: `_compileAddonTemplates`,
+        registry: this.registry,
+      });
+    }
+
+    return tree;
+  }
+
+  _compileAddonJs(tree) {
+    let precompiledSource = new AmdExcluder(tree, {
```

Comment:
lets add a flag to opt-in or out of this tree as an option to `new EmberApp`, the default value should be `true` (e.g. use the amd excluder) but ultimately we should try to change it to `false` once a good number of popular addons have been updated...

Reply:
done